x-pack/plugins/elastic_assistant/server/routes/helpers.ts

```diff
@@ -309,8 +308,9 @@ export const nonLangChainExecute = async ({
 
   telemetry.reportEvent(INVOKE_ASSISTANT_SUCCESS_EVENT.eventType, {
     actionTypeId,
-    isEnabledKnowledgeBase: request.body.isEnabledKnowledgeBase,
-    isEnabledRAGAlerts: request.body.isEnabledRAGAlerts,
+    // TODO: @stephmilovic, what's the right way to go about deprecating these telemetry values?
+    isEnabledKnowledgeBase: true,
+    isEnabledRAGAlerts: true,
```

Comment:
@stephmilovic, recommendation here for sunsetting these telemetry values? Should I just keep sending `true` for now as to not break our dashboards, or actually stop sending them? And if the latter, can we remove them from the telemetry schema now too, or do they need to stick around for awhile?

Reply:
Discussed with Steph, we're good to not send these and remove from the schema 👍